membership-attribute-service/conf/TEST.public.conf

```diff
@@ -1,6 +1,4 @@
 include "DEV.public"
 include "application"
 
-
-
-
+logstash.enabled=true
```

Comment:
Yes, not sure either. It looks like @pvighi added this here: https://github.com/guardian/members-data-api/commit/97f10aa4a93212595a85601e4ab22430f40ba6cb